website/docs/optimizations/java/summary.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Summary"
+sidebar_position: 1000
+---
+
+During this workshop you have built container images with Spring Boot Java Application, applied different optimization technics to container images and deployed container images with Java Application to Amazon EKS cluster.
```

Comment:
During this workshop, you've learned how to create container images using the Spring Boot Java Application. You've also explored various optimization techniques for these container images and successfully deployed them, along with your Java Application, to an Amazon EKS cluster.